doc/espresso-dev-node.md

```diff
@@ -0,0 +1,69 @@
+# Espresso Dev Node
+
+Espresso dev node is a node specifically designed for development and testing. It includes various nodes required to run
+a complete Espresso network, such as `builder`, `sequencer`, etc. Developers can use it for development and testing.
+
+## Download
+
+We highly recommend you to use our Docker image. You can install it from the command line:
+
+```cmd
+docker pull ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:main
+```
+
+## Parameters
+
+| Name                            | Type            | Environment Variable                 | Description                                                                                                                                                             |
```

Comment:
We should also specify what the default behavior is if arguments are not provided.